content/en/docs/components/pipelines/v2/run-a-pipeline.md

```diff
@@ -1,80 +1,107 @@
-client = Client(host='<YOUR_HOST_URL>')
-```
+    <img src="/docs/images/pipelines/start-a-run.png" alt="Start a run screen" class="mt-3 mb-3 border border-info rounded">
 
-To submit IR YAML for execution use the `.create_run_from_pipeline_package` method:
+## Run a pipeline from the KFP SDK client <a id="run_from_sdk_client"></a>
 
-```python
-client.create_run_from_pipeline_package('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
-```
+You can programmatically submit a pipeline run from the KFP SDK client in two ways:
 
-To submit a Python pipeline function for execution use the `.create_run_from_pipeline_func` convenience method, which wraps compilation and run submission into one method:
+* Submit an IR YAML
+* Submit a Python pipeline function for execution
 
-```python
-client.create_run_from_pipeline_func('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
-```
+To submit an IR YAML for execution:
+
+1. Instantiate a `Client` by specifying the `host` URL of your KFP instance:
+    ```python
+    from kfp.client import Client
+    client = Client(host='<YOUR_HOST_URL>')
+    ```
+    For more information about the `Client` constructor and method parameters, see the  [kfp.client][kfp-sdk-api-ref-client] in the [KFP SDK API reference][kfp-sdk-api-ref].
+
+1. Use the `.create_run_from_pipeline_package` method:
+
+    ```python
+    client.create_run_from_pipeline_package('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
+    ```
+
+To submit a Python pipeline function for execution:
+
+1. Instantiate a `Client` by specifying the `host` URL of your KFP instance:
+    ```python
+    from kfp.client import Client
+    client = Client(host='<YOUR_HOST_URL>')
+    ```
+
+1. Use the `.create_run_from_pipeline_func` convenience method. This wraps the compilation and run submission into one method.
+
+    ```python
+    client.create_run_from_pipeline_func('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
+    ```
 
-See the [KFP SDK Client reference documentation][kfp-sdk-api-ref-client] for a detailed description of the `Client` constructor and method parameters.
+## Run a pipeline from the KFP SDK CLI <a id="run_from_sdk_cli">
 
-## 3. Run from the KFP SDK CLI
-The `kfp run create` command allows you to submit a pipeline from the command line. `kfp run create --help` shows that this command takes the form:
+You can submit a pipeline from the KFP SDK CLI using the `kfp run create` command. To run this command, use the following syntax:
 
 ```shell
 kfp run create [OPTIONS] [ARGS]...
 ```
 
-For example, the following command submits the `path/to/pipeline.yaml` IR YAML to the KFP backend:
+For example, run the following command to submit the IR YAML file at `path/to/pipeline.yaml` to the KFP backend:
 
 ```shell
-kfp run create --experiment-name my-experiment --package-file path/to/pipeline.yaml
+kfp run create --experiment-name my-experiment --package-file path/to/pipeline.yaml param="a" other_param=2
```

Comment:
Should we update this example and placeholder style to be consistent with the presentation style from https://github.com/kubeflow/website/pull/3392/files#diff-20420e55f370813ad976788743b9900fa27c72eea0ee22d9a24a72940daf9a4eR114?

Reply:
Discussed offline -- can ignore this comment :)